lib/uplink/bucket_attrs_test.go

```diff
@@ -85,15 +85,17 @@ func TestPartnerBucketAttrs(t *testing.T) {
 			proj.uplinkCfg.Volatile.PartnerID = partnerID
 			got, err := proj.OpenBucket(ctx, bucketName, &access)
 			require.NoError(t, err)
+			assert.Equal(t, got.bucket.Attribution, partnerID)
 
 			info, err := db.Get(ctx, consoleProject.ID, []byte(bucketName))
 			require.NoError(t, err)
 			assert.Equal(t, info.PartnerID.String(), partnerID)
 
-			// partner ID NOT set
+			// partner ID not set but bucket's attribution already set(from above)
 			proj.uplinkCfg.Volatile.PartnerID = ""
 			got, err = proj.OpenBucket(ctx, bucketName, &access)
 			require.NoError(t, err)
+			assert.Equal(t, got.bucket.Attribution, partnerID)
 			defer ctx.Check(got.Close)
```

Comment:
Should the defer be before assert.Equal?

Reply:
moved the defer to before the assert